scene/3d/skeleton_3d.cpp

```diff
@@ -346,11 +346,16 @@ void Skeleton3D::_notification(int p_what) {
 				for (uint32_t i = 0; i < bind_count; i++) {
 					uint32_t bone_index = E->get()->skin_bone_indices_ptrs[i];
 					ERR_CONTINUE(bone_index >= (uint32_t)len);
-					vs->skeleton_bone_set_transform(skeleton, i, bonesptr[bone_index].pose_global * skin->get_bind_pose(i));
+					rs->skeleton_bone_set_transform(skeleton, i, bonesptr[bone_index].pose_global * skin->get_bind_pose(i));
 				}
 			}
 
 			dirty = false;
+
+#ifdef TOOLS_ENABLED
+			emit_signal("pose_updated");
```

Comment:
Emitting a signal here does not make me happy, but I understand why you need it.
Can you replace the C string by a StringName in scene_string_names.h ?

Reply:
Waking, but will make the changes to this when there's a free moment in the evenings.

Reply:
Missed one.

Reply:
Never mind, the create signal uses a string, but the emit signal uses a string name from scene_string_names.h. This is correct.